src/components/VHeader/VSearchBar/VSearchBar.vue

```diff
@@ -12,6 +12,7 @@
       type="search"
       name="q"
       :model-value="searchText"
+      v-on="{ blur: $listeners.blur, focus: $listeners.focus }"
```

Comment:
It seems that there are several unrelated changes and they are causing the app to not work properly. Probably only because of this?

Reply:
Ah sorry @krysal by branch was based on another PR—fixed!